library.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
+    def print_longest_crime_audio_book(self) -> str:
+        crime_audio_books: list[Book] = []
+        max: Optional[Book] = None
+
+        # print banner
+        print("*********************************************")
+        print("********* Longest Crime Audio Book **********")
+        print("*********************************************")
+
+        for b in self.get_books():
+            if Genre.CRIME in b.genres:
+                if b._book_type == "Audio":
+                    if max is not None and max.duration < b.duration:
```

Comment:
As you initialize the `max` with value `None` this condition never compute to true. Please be aware, that there might be audiobooks of same max length. Is it intended that only the first one will be returned? If so, please add this to the documentation of the function.

Reply:
Please be aware, that there might be audiobooks of same max length. Is it intended that only the first one will be returned? If so, please add this to the documentation of the function. Right now you function documentation suggests that multiple books are the return.